app/views/admin/invoices/show.html.erb

```diff
@@ -0,0 +1,36 @@
+<h1> admin invoice show </h1>
+
+<section id="invoice-information">
+  <ul>
+    <li>invoice id: <%= @invoice.id %></li>
+    <li>invoice status: <%= @invoice.status %></li>
+    <li>invoice creation date: <%= @invoice.created_at.strftime("%A, %B %d, %Y") %></li>
+    <li>invoice total: <%= @invoice.invoice_items.invoice_amount %></li>
+  </ul>
+
+  <%= form_with invoice: @invoice, method: :patch do |f| %>
+    <%= f.select :status, Invoice.statuses.keys, selected: "#{@invoice.status}" %>
+    <%= f.label :status %>
+    <%= f.submit %>
+  <% end %>
+</section>
+
+<section id="customer-information">
+  <ul>
+    <li>customer name: <%= @invoice.customer.name %></li>
+    <li>customer address: <%= @invoice.customer.address%></li>
+  </ul>
```

Comment:
We can use method delegation to give the invoice access to its customer's name and address here so that we can call `@invoice.customer_name` and avoid method chaining